aiogram/dispatcher/webhook.py

```diff
@@ -261,7 +261,7 @@ def validate_ip(self):
         if self.request.app.get('_check_ip', False):
             ip_address, accept = self.check_ip()
             if not accept:
-                log.warning(f"Blocking request from an unauthorized IP: {ip_address}")
+                log.warning(f"Blocking request from an unauthorized IP: "+ip_address)
```

Comment:
`ip_address` can be `None`. Best way in this case - use %-style (the same with my previous comment for logging)
```suggestion
                log.warning(f"Blocking request from an unauthorized IP: %s", ip_address)
```